src/main/java/com/android/dexdeps/ClassRef.java

```diff
@@ -18,6 +18,11 @@
 
 import java.util.ArrayList;
 
+/**
+ * This was lifted from the Android source code:
```

Comment:
I'd prefer to leave these files untouched - it's already pretty clear from the source set, `com.android` package, and license header that this is external code.  Let's revert the comments here and in sibling files.